server.go

```diff
@@ -764,7 +768,7 @@ func newServer(cfg *Config, listenAddrs []net.Addr, chanDB *channeldb.DB,
 	s.authGossiper = discovery.New(discovery.Config{
 		Router:            s.chanRouter,
 		Notifier:          s.cc.chainNotifier,
-		ChainHash:         *activeNetParams.GenesisHash,
+		ChainHash:         *s.serverNetParams.GenesisHash,
```

Comment:
Can just use `s.cfg.ActiveNetParams` here and below.

Reply:
yup